{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/conftest.py

```diff
@@ -3,47 +3,37 @@
 import os
```

Comment:
Maybe add a comment to the top of the file that this file is meant in particular for running tests from within the interpreter?

Reply:
Just to be clear it also gets picked up when running vanilla pytest, it just doesn't work right when using tox for now. But I've now added a mention that it needs to be in the package for the interpreter version to work.